docs/decommission.rst

```diff
@@ -0,0 +1,80 @@
+Decommission SecureDrop
+=======================
+
+The following steps will guide you through the decommissioning of your
+SecureDrop instance.
+
+#. **Put a notice in advance on your landing page to inform sources that your
+   instance will soon be retiring.**
+   You may want to direct them to other secure methods of contacting you.
+#. **Locate and create an inventory of all your hardware.**
+   - *Journalist Workstation* USBs
+   - *Admin Workstation* USB(s)
+   - *Secure Viewing Station* USB
+   - *Secure Viewing Station* computer
+   - Transfer and Export devices (USBs, optical drives, or external drives)
+   - Backup USBs/other storage media
+   - Servers
+   - Firewall
+
+   .. note:: The recommended SecureDrop setup includes only one *Secure Viewing
+      Station* USB. However, if you have been working remotely or
+      have a non-standard setup, you may have more than one *SVS USB*. It is
+      important that you locate all of these USBs, since they hold the most
+      sensitive data.
+#. **Decide whether to save a backup.**
+   If you want a backup, you can, follow our `backup guidelines`_. Once the
```

Comment:
"If you want a backup, you can, follow"->"If you want to save a backup, follow"

Reply:
This does not clearly speak to me right now to specific motivations or goals (why would I want to backup, and what steps do I need to follow to do so). One suggestion would be to organize it into two separate steps:

- download any submissions they want to hold on to, and store them in such a manner they'll be able to decrypt/access them securely (e.g., decrypt and store on an encrypted drive, or store alongside Submission Key), then _delete_ all submissions on server (which will trigger SD's own secure deletion);
- if they want to preserve the server configuration for any reason (e.g., may want to keep option open of spining up SD under the same .onion address later), follow the server backup procedures. Otherwise they can skip that step, as long as they've followed the previous one.

For a decommissioning document we can be clear that certain data will be _lost_ if they follow this process, such as any credentials stored on the Admin Workstation USB. I don't think the whole decommissioning process needs to be optionally reversible.

Reply:
Updated in f38127f to reflect, as briefly as possible, the two different paths (server backup/ saving select files), and marked this step as 'optional.' 
As an aside/general point, I'm partly assuming that there will be some support contact before an org decides to follow these steps, so they may not be as fully verbose as some of our other documentation.